packages/reports/src/Helpers/profit-loss.ts

```diff
@@ -0,0 +1,2 @@
+export const getProfitOrLoss = (value: string): 'profit' | 'loss' =>
```

Comment:
```suggestion
export const getProfitOrLoss = (value: string) =>
```
@akmal-deriv TS can infer.